third_party/accessibility/ax/ax_node_data.h

```diff
@@ -274,7 +270,7 @@ struct AX_BASE_EXPORT AXNodeData {
 
   // As much as possible this should behave as a simple, serializable,
   // copyable struct.
-  int32_t id = -1;
+  int32_t id = 0;
```

Comment:
Do we default to zero on other platforms? 0 is always the root node ID in Flutter.

Reply:
e.g. looking at the Android embedder, we default to -1.
https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L1921